### PR TITLE
Docker implementation

### PR DESCRIPTION
**Pull request by enric1994**
Define Dockerized to try it in seconds!

Try: `docker run enric1994/define intelligent`

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM node:10-alpine
+
+WORKDIR /data
+
+RUN apk add --no-cache git
+RUN git clone https://github.com/ziishaned/define-cli.git /data
```
Comment (ziishaned) [inline Dockerfile]: Instead of cloning the repository use `COPY . /data`

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM node:10-alpine
+
+WORKDIR /data
+
+RUN apk add --no-cache git
```
Comment (ziishaned) [inline Dockerfile]: Please remove this

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,9 @@
+FROM node:10-alpine
```
[the remark this answers is not in the document]
Comment (enric1994) [reply Dockerfile]: The version you propose doesn't exist. Try: `docker pull node:latest-alpine`
I used `node:current-alpine` instead

File: Dockerfile (hunk 1 above)
Comment (enric1994) [reply Dockerfile]: Good point. Done

**Comment by ziishaned** [comment]
Thanks

**Pull request merged**